src/db/services/verification.ts

```diff
@@ -0,0 +1,28 @@
+import { Document } from "mongoose";
+
+import { BaseService } from "./base";
+import { Verification } from "../schemas";
+// import { IVerification } from "../types";
+import { VerificationModel } from "../models";
+
+// interface IVerificationService extends IBaseService<Verification> {}
```

Comment:
Does this need to be commented out pt 2 scenes from a memory

Reply:
done